Improve docs to fulfil requirements of bronze quality scale. (#41359)
home-assistant/home-assistant.github.io: source/_integrations/velux.markdown

Rain sensors of supported windows do not report automatically and must be polled every 5 minutes. For this reason, they are disabled by default, because polling uses more radio bandwidth and battery power than simply reporting changed window positions.

> ## Installation preparation

Reviewer: ```suggestion
## Prerequisites
```

> 1. Make sure you have the password for your gateway's wireless access point. You'll find it printed on the underside of your KLF 200 device -- *not* the web login password.

Reviewer: ```suggestion
1. Make sure you have the password for your gateway's wireless access point. 
    - You'll find it printed on the underside of your KLF 200 device.
    - It is not the web login password.
```

> 2. Reboot or power cycle the KLF 200 device. You must complete the configuration within 5 minutes of rebooting the device while its Wi-Fi access point is still visible. 

Reviewer: ```suggestion
2. Reboot or power cycle the KLF 200 device. 
    - You must complete the configuration within 5 minutes of rebooting the device while its Wi-Fi access point is still visible. 
```

> 3. During configuration, keep your Home Assistant connected to your regular network. Don't connect to the device's wireless access point.

Reviewer: ```suggestion
3. During configuration, keep your Home Assistant connected to your regular network. 
    - Don't connect to the device's wireless access point.
```

{% include integrations/config_flow.md %}

During configuration, you will be asked for a hostname and password:

- Hostname: enter the IP address or hostname of the KLF 200 gateway.

> - Password: enter the password of the gateway's wireless access point (printed on the underside - **not** the web login password).

Reviewer: ```suggestion
{% configuration_basic %}
Hostname:
    description: "the IP address or hostname of the KLF 200 gateway. You can find it in your router."
Local access token:
    description: "The password of the gateway's wireless access point. You can find it printed on the underside of the device. It is not the web login password."
{% endconfiguration_basic %}
```
We have this configuration_basic block that you can use for these.

Author: Commitet your suggestion, makes sense, didn't know about the standard blocks. One question though: unless "Local access token" is some kind of standard that's used everywhere, I think "Password" would be a more fitting name.

Reviewer 2: You can keep password there.

Reviewer: > Commitet your suggestion, makes sense, didn't know about the standard blocks. One question though: unless "Local access token" is some kind of standard that's used everywhere, I think "Password" would be a more fitting name.

Thank you for the catch. It should have been "password", of course. That was a copy-and-paste error when I copied from the template.

Author: I suspected this, but wasn't quite sure. Thanks.

Rain sensors of supported windows do not report automatically and must be polled every 5 minutes. For this reason, they are disabled by default, because polling uses more radio bandwidth and battery power than simply reporting changed window positions.

## Prerequisites

1. Make sure you have the password for your gateway's wireless access point.
    - You'll find it printed on the underside of your KLF 200 device.
    - It is not the web login password.
2. Reboot or power cycle the KLF 200 device.
    - You must complete the configuration within 5 minutes of rebooting the device while its Wi-Fi access point is still visible.
3. During configuration, keep your Home Assistant connected to your regular network.
    - Don't connect to the device's wireless access point.

{% include integrations/config_flow.md %}

During configuration, you will be asked for a hostname and password:

{% configuration_basic %}
Hostname:

>     description: "the IP address or hostname of the KLF 200 gateway. You can find it in your router."

Reviewer 2: ```suggestion
    description: "The IP address or hostname of the KLF 200 gateway. You can find it in your router."
```

> Local access token:

Reviewer 2: ```suggestion
Password:
```

In Home Assistant versions up to 2024.12.x, it was recommended to create an automation to reboot the KLF 200 gateway before shutting down or restarting Home Assistant. This automated reboot is now built into the integration, so you no longer need to create a separate automation.

## Removing the integration

This integration follows standard integration removal. No extra steps are required.

> 

Reviewer: ```suggestion

{% include integrations/remove_device_service.md %}
```
This text block adds the actual steps. See also: https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L204 

If there are any other steps needed for integration removal, please add them.
